### ISSUE TITLE
kendo-angular-buttons 5.1.1 gives compile errors

### ISSUE THREAD

**Issue by Matrix159** [telerik/kendo-angular#2667]
**Describe the bug**
The latest 5.1.1 of kendo-angular-buttons breaks angular compiling.

**To Reproduce**
Use 5.1.1 of kendo-angular-buttons to see the errors, downgrade to 5.1.0 to get rid of the errors.

**Kendo packages used**
```
    "@progress/kendo-angular-buttons": "^5.1.0",
    "@progress/kendo-angular-common": "^1.2.1",
    "@progress/kendo-angular-dateinputs": "^4.2.0",
    "@progress/kendo-angular-dropdowns": "4.0.0",
    "@progress/kendo-angular-excel-export": "3.0.0",
    "@progress/kendo-angular-grid": "^4.6.0",
    "@progress/kendo-angular-inputs": "6.1.1",
    "@progress/kendo-angular-intl": "^2.0.0",
    "@progress/kendo-angular-l10n": "^2.0.1",
    "@progress/kendo-angular-layout": "^4.1.5",
    "@progress/kendo-angular-pdf-export": "^2.0.0",
    "@progress/kendo-angular-popup": "^3.0.4",
    "@progress/kendo-angular-tooltip": "^2.1.1",
    "@progress/kendo-data-query": "^1.0.0",
    "@progress/kendo-drawing": "^1.6.0",
    "@progress/kendo-theme-default": "4.1.0",
```
**Errors below**

  ```
  47     click(e: any): void;
         ~~~~~
  ../node_modules/@progress/kendo-angular-grid/dist/es2015/editing/remove-command.directive.d.ts:46:5 - error TS2416: Property 'click' in type 'RemoveCommandDirective' is not assignable to the same property in base type 'ButtonDirective'.
    Type '(e: any) => void' is missing the following properties from type 'EventEmitter<any>': emit, subscribe, observers, closed, and 17 more.
  
  46     click(e: any): void;
         ~~~~~
  ../node_modules/@progress/kendo-angular-grid/dist/es2015/editing/remove-command.directive.d.ts:46:5 - error TS2425: Class 'ButtonDirective' defines instance member property 'click', but extended class 'RemoveCommandDirective' defines it as instance member function.
  
  46     click(e: any): void;
         ~~~~~
  ../node_modules/@progress/kendo-angular-grid/dist/es2015/editing/save-command.directive.d.ts:59:5 - error TS2416: Property 'click' in type 'SaveCommandDirective' is not assignable to the same property in base type 'ButtonDirective'.
    Type '(e: any) => void' is missing the following properties from type 'EventEmitter<any>': emit, subscribe, observers, closed, and 17 more.
  
  59     click(e: any): void;
         ~~~~~
  ../node_modules/@progress/kendo-angular-grid/dist/es2015/editing/save-command.directive.d.ts:59:5 - error TS2425: Class 'ButtonDirective' defines instance member property 'click', but extended class 'SaveCommandDirective' defines it as instance member function.
  
  59     click(e: any): void;
         ~~~~~
  ../node_modules/@progress/kendo-angular-grid/dist/es2015/excel/excel-command.directive.d.ts:33:5 - error TS2416: Property 'click' in type 'ExcelCommandDirective' is not assignable to the same property in base type 'ButtonDirective'.
    Type '(e: any) => void' is missing the following properties from type 'EventEmitter<any>': emit, subscribe, observers, closed, and 17 more.
  
  33     click(e: any): void;
         ~~~~~
  ../node_modules/@progress/kendo-angular-grid/dist/es2015/excel/excel-command.directive.d.ts:33:5 - error TS2425: Class 'ButtonDirective' defines instance member property 'click', but extended class 'ExcelCommandDirective' defines it as instance member function.
  
  33     click(e: any): void;
         ~~~~~
  ../node_modules/@progress/kendo-angular-grid/dist/es2015/pdf/pdf-command.directive.d.ts:33:5 - error TS2416: Property 'click' in type 'PDFCommandDirective' is not assignable to the same property in base type 'ButtonDirective'.
    Type '(e: any) => void' is missing the following properties from type 'EventEmitter<any>': emit, subscribe, observers, closed, and 17 more.
  
  33     click(e: any): void;
         ~~~~~
  ../node_modules/@progress/kendo-angular-grid/dist/es2015/pdf/pdf-command.directive.d.ts:33:5 - error TS2425: Class 'ButtonDirective' defines instance member property 'click', but extended class 'PDFCommandDirective' defines it as instance member function.
  
  33     click(e: any): void;
         ~~~~~
```

**Comment by clabough** [comment]
Same here.

**Comment by andrpelz** [comment]
Have the same problem with kendo-angular-buttons version 5.1.1

**Comment by valchev** [comment]
There was an issue caused by the latest version of the Buttons package that leads to conflicting names when buttons are used within the Grid. We addressed it immediately and released a 4.6.1 version of the Grid package that should resolve the problem:

https://www.telerik.com/kendo-angular-ui-develop/components/grid/changelog/

Upgrading to the latest Grid version should resolve the problem.

I apologize for the caused inconvenience.

**Comment by clabough** [comment]
works for me now. thanks
